src/Kovan-DssSpell.t.sol

```diff
@@ -1321,6 +1321,9 @@ contract DssSpellTest is DSTest, DSMath {
                     (exists, value) = clip.calc().call(abi.encodeWithSignature("cut()"));
                     uint256 normalizedTestCut = values.collaterals[ilk].calc_cut * 10**23;
                     assertEq(exists ? abi.decode(value, (uint256)) : 0, normalizedTestCut);
```

Comment:
Might as well move these statements inside the `if` block too; can eliminate the ternary operator in the assertion then.

Reply:
Thought about it, but as the other two were doing the same way I didn't want to change it. And btw that would change 1 line to 2 lines of code hehe